docs-wiki/branching.md

```diff
@@ -23,11 +23,28 @@ gitGraph
 
 On this page:
 
+- [🌟 Important branches](#-important-branches)
 - [💡 Tips for external contributors](#-tips-for-external-contributors)
 - [💡 Tips for Microsoft contributors](#-tips-for-microsoft-contributors)
 
 ---
 
+## 🌟 Important branches
+
+- `main` is for released versions. Do not work in this branch.
+- `dev` is for the next release. All PRs to this branch must meet the following requirements:
+  - Changes must be complete and validated. No partial commits.
+  - Applicable documentation in [docs](../docs) must be updated.
+  - External-facing changes must be covered in the [changelog](../docs/changelog.md).
```

Comment:
Need to confirm once published to Wiki - don't think relative links will work like this.

Reply:
Yeah, you're right. They need to be `../tree/main/docs`. Thanks!

Reply:
Yeah, you're right. They need to be `../tree/dev/docs`. Thanks!
```suggestion
  - External-facing changes must be covered in the [changelog](../tree/dev/docs/changelog.md).
```